build/media-suite_compile.sh

```diff
@@ -1992,30 +1992,30 @@ if [[ $mpv != "n" ]] && pc_exists libavcodec libavformat libswscale libavfilter;
 
     _check=(libvulkan.a vulkan.pc)
     if ! mpv_disabled vulkan &&
-        do_vcs "https://github.com/KhronosGroup/Vulkan-Loader.git#tag=v1.1.114" vulkan-loader; then
-        _shinchiro_patches="https://raw.githubusercontent.com/shinchiro/mpv-winbuild-cmake/b1f8bc13/packages"
+        do_vcs "https://github.com/KhronosGroup/Vulkan-Loader.git" vulkan-loader; then
+        _DeadSix27="https://raw.githubusercontent.com/DeadSix27/python_cross_compile_script/master"
         do_uninstall "${_check[@]}"
-        do_patch "$_shinchiro_patches/vulkan-0001-cross-compile-static-linking-hacks.patch"
+        do_patch "$_DeadSix27/patches/vulkan/0001-fix-cross-compiling.patch"
         create_build_dir
         log dependencies /usr/bin/python3 ../scripts/update_deps.py --no-build
         cd_safe Vulkan-Headers
-            _check=(vulkan/vulkan.h)
-            do_uninstall include/vulkan
+            _check=(vulkan/vulkan.h d3d{kmthk,ukmdt}.h)
+            do_uninstall include/vulkan d3d{kmthk,ukmdt}.h
             do_cmakeinstall
+            do_wget -c -r -q "$_DeadSix27/additional_headers/d3dkmthk.h"
+            do_wget -c -r -q "$_DeadSix27/additional_headers/d3dukmdt.h"
+            do_install d3d{kmthk,ukmdt}.h include/
             do_checkIfExist
             _check=(libvulkan.a vulkan.pc)
         cd_safe "$LOCALBUILDDIR/$(get_first_subdir)"
-        CFLAGS+=" -D_WIN32_WINNT=0x0600 -D__STDC_FORMAT_MACROS" \
-            CPPFLAGS+=" -D_WIN32_WINNT=0x0600 -D__STDC_FORMAT_MACROS" \
-            CXXFLAGS+=" -D__USE_MINGW_ANSI_STDIO -D__STDC_FORMAT_MACROS -fpermissive -D_WIN32_WINNT=0x0600" \
```

Comment:
These Cflags aren't needed anymore?

Reply:
The default of _WIN32_WINNT is [0x0601](https://github.com/msys2/MINGW-packages/blob/master/mingw-w64-headers-git/PKGBUILD#L58)
__USE_MINGW_ANSI_STDIO is already defined in our batch file
__STDC_FORMAT_MACROS is [not needed](https://en.cppreference.com/w/cpp/types/integer#Notes) if you have c++11
The CMakeLists sets `-std=c++11` [here](https://github.com/KhronosGroup/Vulkan-Loader/blob/160e66b55a385eec961187b6e422417541c25608/CMakeLists.txt#L150)
`-fpermissive` is not needed and shouldn't be used IMO